config/config_tests.xml

```diff
@@ -354,7 +357,7 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <DOUT_S>FALSE</DOUT_S>
   </test>
 
-  <test NAME="TESTBUILDFAIL">
+  <test NAME="TESTBUILDFAIL" INFRASTRUCTURE_TEST="TRUE">
```

Comment:
I don't know if this was the right way to do this. I wanted some way to exclude these infrastructure-only tests, because otherwise they clutter the output. I'm open to suggestions of a better way to denote this.

Reply:
Seems like a decent approach to me. The other way would be check if the test is a subclass of FakeTest.